client/dashboard/sites/scan/notices.tsx

```diff
@@ -0,0 +1,86 @@
+import { JETPACK_CONTACT_SUPPORT } from '@automattic/urls';
+import { Button, ExternalLink } from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { useState, useEffect } from 'react';
+import { Notice } from '../../components/notice';
+import type { ScanStatusType } from './use-scan-state';
+
+interface ScanNoticesProps {
+	status: ScanStatusType;
+	threats: number;
+}
+
+export function ScanNotices( { status, threats }: ScanNoticesProps ) {
+	const [ isDismissed, setIsDismissed ] = useState( false );
+
+	const handleDismiss = () => {
+		setIsDismissed( true );
+	};
+
+	useEffect( () => {
+		// Reset dismissal when a new scan starts
+		if ( status === 'running' ) {
+			setIsDismissed( false );
+		}
+	}, [ status ] );
+
+	function NoticeError() {
+		return (
+			<Notice
+				variant="error"
+				title={ __( 'Latest scan for your site couldn’t be completed' ) }
+				onClose={ handleDismiss }
+				actions={
+					<Button variant="primary" href={ JETPACK_CONTACT_SUPPORT } target="_blank">
+						{ __( 'Contact support' ) }
+					</Button>
+				}
+			>
+				{ createInterpolateElement(
+					__(
+						'Please check to see if your site is down – if it’s not, try again. If it is, or if the scan process is still having problems, <external>check our help guide</external> or contact our support team.'
+					),
+					{
+						// @ts-expect-error children prop is injected by createInterpolateElement
+						external: <ExternalLink href="https://jetpack.com/support/scan/" />,
```

Comment:
We could pass `children={ null }` to avoid using the `@ts-expect-error`

source: p1757330597241709/1757329266.026149-slack-C0982082MSR